packages/components/select-v2/src/defaults.ts

```diff
@@ -13,8 +19,9 @@ import { defaultProps } from './useProps'
 import type { Option, OptionType } from './select.types'
 import type { Props } from './useProps'
 import type { Options, Placement } from '@element-plus/components/popper'
+import type { EmitFn } from '@element-plus/utils/vue/typescript'
 
-export const SelectProps = buildProps({
+export const selectProps = buildProps({
```

Comment:
It may be better to keep the first letter capitalized here, otherwise it will be a breaking change for users who use related values.

Reply:
You are right.